examples/torch/common/models/classification/efficientnet.py

```diff
@@ -23,5 +23,8 @@
 
 def efficient_net(pretrained=True, num_classes=1000, **kwargs):
     if pretrained:
-        return EfficientNet.from_pretrained(num_classes=num_classes, **kwargs)
-    return EfficientNet.from_name(num_classes=num_classes, **kwargs)
+        model = EfficientNet.from_pretrained(num_classes=num_classes, **kwargs)
+    else:
+        model = EfficientNet.from_name(num_classes=num_classes, **kwargs)
+    model.set_swish(memory_efficient=False)
```

Comment:
It breaks backward compatibility with previous interface. That's why tests for searching blocks fail. 
Propose to call `set_swish` after model creation when it's needed, and not do it by default on creation.

Reply:
Thanks! I reverted this change and I am calling `set_swish` after model creation.